zcash_primitives/src/transaction/sighash.rs

```diff
@@ -338,7 +341,6 @@ pub fn signature_hash_data<'a>(
                     script_code,
                     value,
                 } => {
-                    #[cfg(feature = "zfuture")]
```

Comment:
Why was this removed? This code shouldn't compile without the feature flag anymore (which is the opposite of what the commit says it is fixing).